### PR TITLE
refactor: add missing config for jaeger

### PR DESCRIPTION
**Pull request by daviddelucca**
Added missing configuration for jaeger remote sampler.

Without this configuration, traces are not being sent to Jaeger.

## Related Issue or Design Document

https://github.com/ory/x/issues/655

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: otelx/config.schema.json [hunk 1]
```diff
@@ -56,6 +56,11 @@
                   "description": "The address of jaeger-agent's HTTP sampling server",
                   "format": "uri",
                   "examples": ["http://localhost:5778/sampling"]
+                },
+                "trace_id_ratio": {
```
Comment (alnr) [inline otelx/config.schema.json:60]: I believe this needs to be listed in line 50 as well.

File: otelx/jaeger.go [hunk 2]
```diff
@@ -50,6 +50,7 @@ func SetupJaeger(t *Tracer, tracerName string, c *Config) (trace.Tracer, error)
 		jaegerRemoteSampler := jaegerremote.New(
 			"jaegerremote",
 			jaegerremote.WithSamplingServerURL(samplingServerURL),
+			jaegerremote.WithInitialSampler(sdktrace.TraceIDRatioBased(c.Providers.Jaeger.Sampling.TraceIdRatio)),
```
Comment (alnr) [inline otelx/jaeger.go:53]: Can you add this to `otelx/config_test.go`, please?

**Review by alnr — CHANGES_REQUESTED** [review]
Thank you!

File: otelx/config.schema.json (hunk 1 above)
Comment (daviddelucca) [reply otelx/config.schema.json:60]: Oops! I forgot to commit this file 🙈

File: otelx/jaeger.go (hunk 2 above)
Comment (daviddelucca) [reply otelx/jaeger.go:53]: Done!

**Comment by alnr** [comment]
Beautiful!

**Review by alnr — APPROVED** [review]

**Review by aeneasr — APPROVED** [review]

**Pull request merged**